### DISCUSSION TITLE
Possible to use a private npm repository for LSPs etc?

### DISCUSSION THREAD

**Discussion by benfrain** [zed-industries/zed#7976] [Feature Requests]
At work, our InfoSec prevents access to npm. As such we have a private repository using ProGet that sits as our own npm.

Is it possible configure Zed to look to somewhere other than npm for language server installs?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
